### ISSUE TITLE
Add buildType and flavor to output APK file name.

### ISSUE THREAD

**Issue by Glennmen** [SUPERCILEX/version-orchestrator#5]
Currently this plugin removes the default APK output naming behaviour.
By default Gradle adds buildtype and flavor names to the build APK output name.

Example: `com.example-basic-release.apk`
`com.example` -> `archivesBaseName`
`basic` -> `productFlavors`
`release` -> `buildTypes`

Also it works with multiple flavors: `com.example-basic-free-release.apk`
`com.example` -> `archivesBaseName`
`basic` -> `1st dimension productFlavors`
`free` -> `2nd dimension productFlavors`
`release` -> `buildTypes`

Can this be added back? As we have lots of different flavours it will help us a lot to have this information in the APK file name.

**Comment by SUPERCILEX** [comment]
Do you mean the name should include the info above *and* the git describe details? That sounds like a good idea. Otherwise, you can just disable name configuration: https://github.com/SUPERCILEX/version-master#disabling-version-name-configuration.

**Comment by Glennmen** [comment]
Yes that is what I meant, it would be great to have the output filename contain flavours and buildtype.
I am only talking about the output APK filename, not the version name.

**Comment by SUPERCILEX** [comment]
Done! I put everything you could ever want in the file name:
```
issue-mcve-com.supercilex.test-flavor-release-0.1.0-5-g83c80a1-dirty.apk
```

**Comment by SUPERCILEX** [comment]
Shipped in v0.6.0.

**Comment by Glennmen** [comment]
This was 100% what I was expecting! Thank you for the quick update.
